sonnet/src/mixed_precision.py

```diff
@@ -77,21 +77,23 @@ def _cast_call(f, new_dtype, args, kwargs):
 def modes(valid_types):
   """Decorate a function to cast inputs/outputs to different precision.
 
-      snt.Linear.__call__ = snt.mixed_precision.modes(
-        [tf.float32, tf.float16])(snt.Linear.__call__)
-      mod = snt.Linear(10)
-      snt.mixed_precision.enable(tf.float16)
-      y = mod(tf.ones([1, 1]))  # First call will be done in F32.
-      y = mod(tf.ones([1, 1]))  # MatMul/Add will be done in F16.
+      .. code-block:: python
+
+          snt.Linear.__call__ = snt.mixed_precision.modes([tf.float32, tf.float16])(snt.Linear.__call__)
+          mod = snt.Linear(10)
+          snt.mixed_precision.enable(tf.float16)
+          y = mod(tf.ones([1, 1]))  # First call will be done in F32.
+          y = mod(tf.ones([1, 1]))  # MatMul/Add will be done in F16.
 
   Args:
     valid_types: Collection of types that the function being decorated is legal
-      to run in.
+    to run in.
 
   Returns:
     A decorator that will cast the inputs and outputs of the decorated function
     according to the global mixed precision policy and the functions eligibility
     for mixed precision.
+
```

Comment:
This is an extra newline, right? If so, could you remove it please? Otherwise it might fail our internal linter checks.